data/cloudcraft.json

```diff
@@ -1751,6 +1751,167 @@
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "The name of the team.",
+                      "example": "Example Team Name"
+                    },
+                    "visible": {
+                      "type": "boolean",
+                      "description": "Whether the team is visible to other users.",
+                      "example": true
+                    },
+                    "crossOrganizational": {
+                      "type": "boolean",
+                      "description": "Whether the team is cross-organizational.",
+                      "example": false
+                    },
+                    "customerId": {
+                      "type": "string",
+                      "description": "The unique customer ID of the team.",
+                      "example": "86e9951e-3764-48d7-b318-969f8a6d180b"
+                    },
+                    "updatedAt": {
+                      "type": "string",
+                      "description": "The date and time the team was last updated.",
+                      "example": "2022-01-01T20:54:53.963Z"
+                    },
+                    "createdAt": {
+                      "type": "string",
+                      "description": "The date and time the team was created.",
+                      "example": "2022-01-01T20:54:47.282Z"
+                    },
+                    "externalSharing": {
+                      "type": "boolean",
+                      "description": "Whether external sharing is enabled for the team.",
+                      "example": true
+                    },
+                    "role": {
+                      "type": "string",
+                      "description": "The role of the user in the team.",
+                      "example": "admin"
+                    },
+                    "members": {
+                      "type": "object",
+                      "description": "An array of key-value pairs representing the team's members.",
+                      "properties": {
+                        "key": {
+                          "type": "string",
+                          "example": "value"
+                        }
+                      }
+                    }
+                  }
+                },
+                "examples": {
+                  "Get team information": {
+                    "value": {
+                      "id": "us46e9aa-5806-4cd6-8e78-c22d58602d09",
+                      "name": "Example Team Name",
+                      "visible": true,
+                      "crossOrganizational": false,
+                      "customerId": "86e9951e-3764-48d7-b318-969f8a6d180b",
+                      "updatedAt": "2022-01-01T20:54:53.963Z",
+                      "createdAt": "2022-01-01T20:54:47.282Z",
+                      "externalSharing": true,
+                      "role": "admin",
+                      "settings": {
```

Comment:
This `settings` key looks out of place here 🤔

Reply:
Ops, copy/paste error. Fixed with [01ac2b2](https://github.com/DataDog/documentation/pull/22997/commits/01ac2b20fdad7269c916b966b0cf4e5c825cd4f8).